src/raft/uv_segment.c

```diff
@@ -748,12 +749,12 @@ int uvSegmentBufferAppend(struct uvSegmentBuffer *b,
 	int rv;
 
 	size = sizeof(uint32_t) * 2;            /* CRC checksums */
-	size += uvSizeofBatchHeader(n_entries, true); /* Batch header */
+	size += uvSizeofBatchHeader(n_entries, format_version); /* Batch header */
 	for (i = 0; i < n_entries; i++) {       /* Entries data */
 		size += bytePad64(entries[i].buf.len);
-#ifdef DQLITE_NEXT
-		size += sizeof(struct raft_entry_local_data);
-#endif
+		if (format_version == RAFT_UV_FORMAT_V2) {
+			size += sizeof(struct raft_entry_local_data);
```

Comment:
Why is it that we use `sizeof(struct raft_entry_local_data)` here and in `src/raft/uv_encoding.c` we hardcode 8?

Reply:
I think the patch uses the symbolic form of `sizeof(struct raft_entry_local_data)` (= 16) consistently---the 8s for `sizeof(uint64_t)` are mostly pre-existing and I was trying to keep the patch focused.

Reply:
Ah I see. I was not suggesting to refactor, just asking why one is different from the other 16 != 8.